cvat-ui/src/reducers/index.ts

```diff
@@ -263,6 +263,11 @@ export type PluginsList = {
     [name in SupportedPlugins]: boolean;
 };
 
+export interface PluginsRegistrar {
+    modelsPlugin: () => void;
```

Comment:
Does core app know about models plugin?

Reply:
No, it's core app plugin that helps cvat-ui to react on LambdaManager model list changes. Core app doesnt know when and how this list changes. Thats how lambda functions and functions from external providers are getting mixed in one list

Reply:
Need to discuss it